_benchmark/workloads/vectorsearch.md

```diff
@@ -0,0 +1,572 @@
+|                                   50th percentile service time | custom-vector-bulk |     36.3095 |     ms |
+|                                   90th percentile service time | custom-vector-bulk |     52.2662 |     ms |
+|                                  100th percentile service time | custom-vector-bulk |     68.6513 |     ms |
+|                                                     error rate | custom-vector-bulk |           0 |      % |
+|                                                 Min Throughput |       prod-queries |      211.26 |  ops/s |
+|                                                Mean Throughput |       prod-queries |      213.85 |  ops/s |
+|                                              Median Throughput |       prod-queries |      213.48 |  ops/s |
+|                                                 Max Throughput |       prod-queries |      216.49 |  ops/s |
+|                                        50th percentile latency |       prod-queries |     3.43393 |     ms |
+|                                        90th percentile latency |       prod-queries |     4.01881 |     ms |
+|                                        99th percentile latency |       prod-queries |     5.56238 |     ms |
+|                                      99.9th percentile latency |       prod-queries |     9.95666 |     ms |
+|                                     99.99th percentile latency |       prod-queries |     39.7922 |     ms |
+|                                       100th percentile latency |       prod-queries |      62.415 |     ms |
+|                                   50th percentile service time |       prod-queries |     3.43405 |     ms |
+|                                   90th percentile service time |       prod-queries |      4.0191 |     ms |
+|                                   99th percentile service time |       prod-queries |     5.56316 |     ms |
+|                                 99.9th percentile service time |       prod-queries |     9.95666 |     ms |
+|                                99.99th percentile service time |       prod-queries |     39.7922 |     ms |
+|                                  100th percentile service time |       prod-queries |      62.415 |     ms |
+|                                                     error rate |       prod-queries |           0 |      % |
+
+
+---------------------------------
+[INFO] SUCCESS (took 119 seconds)
+---------------------------------
+```
+
+### Faiss results
+
+The following sample outputs are from the Faiss IVF benchmarking procedure. For brevity, the test used 100 search queries instead of the 10,000 specified in the parameter files. All other parameters remain the same as those in the `params/train` folder. The first run demonstrates results without quantization, the second run shows scalar quantization, and the third run illustrates product quantization. Note that quantization may cause search recall to drop.
```

Comment:
```suggestion
The following sample outputs were generated using the Faiss IVF benchmarking procedure. For brevity, the test used 100 search queries instead of the 10,000 specified in the parameter files. All other parameters remain the same as those in the `params/train` folder. The first run demonstrates results without quantization, the second run demonstrates scalar quantization, and the third run demonstrates product quantization. Note that quantization may cause search recall to drop.
```